vectorization/reshape_vectors.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+
+def reshape_vectors(vectors):
```

Comment:
When is this function used? At the moment it is not very obvious (due to the lack of structure in the main and the necessity to introduce manual changes). Either logically unite it with some of the existing classes, or write docstring